UndertaleModTool/Program.cs

```diff
@@ -37,7 +37,7 @@ public static void Main()
             catch (Exception e)
             {
                 File.WriteAllText(Path.Combine(GetExecutableDirectory(), "crash.txt"), e.ToString());
-                MessageBox.Show(e.ToString());
+                MessageBox.Show(Application.Current.MainWindow as MainWindow, e.ToString());
```

Comment:
Clarifying mainwindow as a parent here can cause issues if the mainwindow fails to start.